src/pages/HomePage.tsx

```diff
@@ -0,0 +1,16 @@
+import { WrappedDocument } from "@govtechsg/open-attestation";
```

Comment:
rename it to home.tsx (page is redundant, let's follow https://google.github.io/styleguide/jsguide.html#file-name and uses dashes)

Can you rename all the files of that MR ?